tests/Models/BeatmapsetRequalifyTest.php

```diff
@@ -262,6 +262,11 @@ protected function setUp(): void
         Bus::fake([CheckBeatmapsetCovers::class]);
     }
 
+    private function assertDiffWithinRange(int $expected, int $actual, int $range = 1)
```

Comment:
shouldn't return anything? also the function call reads kinda weird with the default